### DISCUSSION TITLE
why result no json

### DISCUSSION THREAD

**Discussion by osama300** [flutter-form-builder-ecosystem/form_builder_file_picker#64] [Q&A] [answered]
how i can read it as json
{images: [PlatformFile(, name: user.json, bytes: [11, 44, 398, 101, 114, 34, 555, 52, 34, 125], readStream: null, size: 132)]}

**Comment by deandreamatias** [comment]
I don't understanding your question. Please explain more about it

**Reply by osama300** [reply]
when i click submit i get this : 
{images: [PlatformFile(, name: user.json, bytes: [11, 44, 398, 101, 114, 34, 555, 52, 34, 125], readStream: null, size: 132)]}

but it is not json format i want json

**Reply by deandreamatias** [reply]
OK. What do you want on json? What data?

**Reply by osama300** [reply]
this correct json : {
  "images": [
    {
      "name": "user.json",
      "bytes": [11, 44, 398, 101, 114, 34, 555, 52, 34, 125],
      "readStream": null,
      "size": 132
    }
  ]
}

**Reply by deandreamatias** [reply, marked as answer]
OK,nice. Until I review the code to include this, you can use the transformValue, I guess
